packages/medusa-js/src/request.ts

```diff
@@ -190,9 +190,9 @@ class Client {
       headers: this.setHeaders(options, method, path),
     }
 
```

Comment:
Nit: maybe it is just me but seems there is a bit of mental gymnastics involved in understanding what is happening here; maybe add:

```
    // e.g. data = { cart: { .. } }, response = { status, headers, ... }
    const { data, ...response } = await this.axiosClient(reqOpts)
    
    // e.g. return object = { cart, response }
    return { ...data, response }
```